### PR TITLE
Wording improvements to bg.po

### PR DESCRIPTION
**Pull request by RacerBG**
I already made this changes in Transifex. If no further changes are made to the strings in the 1.7.x branch - I believe this should be the final version of my Bulgarian translation.

Merry Christmas!

### CODE REVIEW

**Comment by levinli303** [comment]
Merry Christmas. I think there are some CI issues that need to be resolved before we can merge this.

**Comment by RacerBG** [comment]
> 
> 
> Merry Christmas. I think there are some CI issues that need to be resolved before we can merge this.

I see but how I can do this? Unfortunately I'm not using git - I'm just using the GitHub infrastructure.

**Comment by levinli303** [comment]
@RacerBG you don't need to do anything. we need to merge this https://github.com/CelestiaProject/Celestia/pull/900 first

**Comment by RacerBG** [comment]
> 
> 
> @RacerBG you don't need to do anything. we need to merge this #900 first

That's a big relief, thank you very much @levinli303 !

File: po/bg.po [hunk 1]
```diff
@@ -3,15 +3,16 @@
 # 
 # Translators:
 # Hleb Valoshka <375gnu@gmail.com>, 2020
-# Georgi Georgiev <georgiev_1994@abv.bg>, 2020
+# Georgi Georgiev (Жоро) <g.georgiev.shumen@gmail.com>, 2020
 # 
+#, fuzzy
```
Comment (375gnu) [inline po/bg.po:8]: this line should be removed

**Review by 375gnu — CHANGES_REQUESTED** [review]

**Review by 375gnu — CHANGES_REQUESTED** [review]
please merge commits into a single one

**Comment by RacerBG** [comment]
> 
> 
> please merge commits into a single one

@375gnu Any way to do that from the site interface, I don't have git?

**Comment by 375gnu** [comment]
unfortunately without local `git` this is not possible

**Comment by RacerBG** [comment]
> 
> 
> unfortunately without local `git` this is not possible

Looks like the only feasible solution then is to create this PR again (with everything included) and to close the current one.

**Comment by RacerBG** [comment]
Superseded by #903 .

**Pull request closed without merging**